app/database/seeds/01_clients.js

```diff
@@ -0,0 +1,30 @@
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('clients').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('clients').insert([
+        {
+          id: 1,
```

Comment:
I think each clients also needs a foreign key to the jobs table

Reply:
Added!